src/cbmc/cbmc_parse_options.cpp

```diff
@@ -972,6 +978,7 @@ void cbmc_parse_optionst::help()
     " --xml-ui                     use XML-formatted output\n"
     " --xml-interface              bi-directional XML interface\n"
     " --json-ui                    use JSON-formatted output\n"
+    " --validate-goto-model        enables additional well-formedness checks on the goto program\n" // NOLINT(*)
```

Comment:
break long help text and align below `enables`

Reply:
You mean so the output of `--help` will be at most 80 characters wide, or that the code should be at most 80 characters wide, or both?

Reply:
https://github.com/diffblue/cbmc/blob/7905492ffce5c0f11405474d11cc673c8f02b55f/src/goto-instrument/reachability_slicer.h#L44-L47

Reply:
Thanks @xbauch. So that seems to suggest 80 characters in the output but not necessarily in the code. I'm gonna change it to that.